### PR TITLE
update Binomial regression notebook to v4

### PR DESCRIPTION
**Pull request by drbenvincent**
#214 

These changes _should hopefully_ bring this notebook up to v4 spec. Just let me know if there's anything amiss.

### CODE REVIEW

File: examples/references.bib [hunk 1]
```diff
@@ -366,3 +366,10 @@ @book{wilkinson2005grammar
   issn={1431-8784},
   isbn={978-0-387-24544-7}
 }
+
+@book{roback2021beyond,
+  title={Beyond multiple linear regression: Applied generalized linear models and multilevel models in R},
+  author={Roback, P., and Legler, J.},
+  year={2021},
+  publisher={CRC Press}
```
Comment (OriolAbril) [inline examples/references.bib:374]: you can also add a `url` field to include https://bookdown.org/roback/bookdown-BeyondMLR/ in the bibliography citation too.

**Review by OriolAbril — COMMENTED** [review]
commented two nits, the main issue are the git conflicts with the references file

**Comment by drbenvincent** [comment]
<p>Fixed in an imminent commit</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/273/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by drbenvincent** [comment]
<p>Thanks, this is handy to know! Still learning about the wonders of Xarray. Have replaced this in an imminent commit</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/273/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: examples/references.bib (hunk 1 above)
Comment (drbenvincent) [reply examples/references.bib]: Done :)

**Comment by drbenvincent** [comment]
Made those changes 👍

I'm not sure why I'm getting conflicts with the references.bib. Do I need to do a rebase before I create a new feature branch?

**Comment by OriolAbril** [comment]
The bibliography is not working, it looks like the whole directive is in a single line, it must be in 3 lines minimum like in the example at https://docs.pymc.io/en/latest/contributing/jupyter_style.html#references

**Review by OriolAbril — APPROVED** [review]

**Review by ltoniazzi — COMMENTED** [review]
Two tiny comments: (i) the [Jupyter style guide](https://docs.pymc.io/en/latest/contributing/jupyter_style.html) suggests to not use unicode in code blocks; (ii) Successes is sometimes mis-spelled

**Comment by drbenvincent** [comment]
Thanks @ltoniazzi. I'm not a fan of the no unicode thing, but I've done that.

I've also added `# hide-input` comment at the start of two long cells of plotting code. This ability to have some [code cells collapsed](https://myst-nb.readthedocs.io/en/latest/use/hiding.html) is really neat and show allow much more focus on the concepts.

**Comment by drbenvincent** [comment]
I'm running the pre-commit checks, but the new `bibtex-tidy` is showing up as skipped. So not sure what to do about that
Screenshot 2022-02-06 at 11 30 09

**Comment by OriolAbril** [comment]
Try running `pre-commit run --all`, then commititng the changes. I think there is also a github command/shortcut but I never remember it.

**Comment by drbenvincent** [comment]
All checks have passed 🥳

**Review by michaelosthege — APPROVED** [review]
Looks good to me!

Also TIL that `arviz.plot_hdi` is an alternative/addition to `pm.gp.util.plot_gp_dist`.
On that side note: @OriolAbril what do you think about migrating `pm.gp.util.plot_gp_dist` to arviz and extending it with a kwarg to draw HDI limits kind of like this?
grafik

Shall I open an Issue/Discussion over at ArviZ ?

**Comment by OriolAbril** [comment]
I hardly ever use gps and don't know what `pm.gp.util.plot_gp_dist` is, but an issue/discussion sounds good. do you know why hasn't it been moved yet? does it rely heavily on the model or extra metadata not present in inferencedata?

**Pull request merged**